src/axom/inlet/Table.hpp

```diff
@@ -277,52 +291,82 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
    * \return Shared pointer to the created Field
    *****************************************************************************
    */
-  std::shared_ptr<Table> addStringArray(const std::string& name,
-                                        const std::string& description = "");
-
-  /*!
-   *****************************************************************************
-   * \brief Get a boolean array represented as an unordered map from the input deck
-   *
-   * \param [out] map Unordered map to be populated with array contents
-   *
-   * \return Whether or not the array was found
-   *****************************************************************************
-   */
-  bool getArray(std::unordered_map<int, bool>& map);
+  std::shared_ptr<Verifiable> addStringArray(
+    const std::string& name,
+    const std::string& description = "");
 
   /*!
    *****************************************************************************
-   * \brief Get a int array represented as an unordered map from the input deck
+   * \brief Add an array of Fields to the input deck schema.
    *
-   * \param [out] map Unordered map to be populated with array contents
+   * \param [in] name Name of the array
+   * \param [in] description Description of the Field
    *
-   * \return Whether or not the array was found
+   * \return Shared pointer to the created Field
    *****************************************************************************
    */
-  bool getArray(std::unordered_map<int, int>& map);
+  std::shared_ptr<Table> addGenericArray(const std::string& name,
+                                         const std::string& description = "");
 
   /*!
    *****************************************************************************
-   * \brief Get a double array represented as an unordered map from the input deck
+   * \brief Get an array represented as an unordered map from the input deck
```

Comment:
This keeps cropping up from random sources but can you change `input deck` to `input file`